rest-api/test/unit_test/api_test/awardee_api_test.py

```diff
@@ -48,6 +48,17 @@ def setUp(self):
     hpo_dao.insert(HPO(hpoId=AZ_HPO_ID, name='AZ_TUCSON', displayName='Arizona',
                        organizationType=OrganizationType.HPO))
 
+  def test_active_and_inactive_site(self):
+    self._setup_data()
+    result = self.send_get('Awardee')
```

Comment:
We should assert the full contents of these responses, not just that the first site is active or inactive. See below.

Reply:
mostly done 😉

Reply:
OK, I'm not seeing that yet. Are your changes committed?

I was expected make_expected_pitt_awardee_resource to start taking a flag...

Reply:
done

Reply:
So lines 54 - 62 here aren't asserting the full contents of the response. Can we instead do like below:

 self.assertEquals(_make_awardee_with_resource(self._make_expected_pitt_awardee_resource(
                                                   inactive=False), 'PITT'),  result);
                                                   result['entry'][1])

Reply:
done